injuries/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "cu-ui-injuries",
+  "version": "0.1.0",
+  "description": "Camelot Unchained UI - injuries",
+  "license": "MPL-2.0",
+  "main": "src/ts/main.tsx",
+  "contributors": [
+    "mehuge@sorcerer.co.uk"
+  ],
+  "keywords": [
+    "Camelot",
+    "Unchained",
+    "cu",
+    "cu-ui"
+  ],
+  "files": [
+    "lib/",
+    "package.json",
+    "README.md",
+    "LICENSE"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/csegames/cu-ui.git"
+  },
+  "dependencies": {
+    "cu-components": "^0.1.3",
+    "cu-core": "^0.1.12",
+    "cu-events": "^0.1.6",
+    "cu-stores": "^0.1.5",
+    "react": "^0.13.3",
+    "reflux": "^0.3.0"
+  },
+  "devDependencies": {
+    "gulp": "^3.9.0",
+    "cu-build-tools": "^0.1.8"
+  }
+}
```

Comment:
"private": false

We want to set our modules to private modules in our package.json files so that it does not get accidentally uploaded to npmjs.org.  We'll have to make a note to change this in our boilerplate as well. !fix  !fix